htdocs/core/lib/pdf.lib.php

```diff
@@ -1077,7 +1077,7 @@ function pdf_writelinedesc(&$pdf,$object,$i,$outputlangs,$w,$h,$posx,$posy,$hide
  *  @param  int			$issupplierline      Is it a line for a supplier object ?
  *  @return string       				     String with line
  */
-function pdf_getlinedesc($object,$i,$outputlangs,$hideref=0,$hidedesc=0,$issupplierline=0)
+function pdf_getlinedesc($object,$i,$outputlangs,$hideref=0,$hidedesc=0,$issupplierline=0,$showrefcustomerinsupplierline=0)
```

Comment:
The supplier order is normally printed prior to be sent to the supplier. There is generally no need to print this document for own use, whatever I provide the parameter to restore this behaviour. Perhaps should I provide also a database constant to set $showrefinsupplierline. I inverted the default behaviour as I disliked the  appearance of line labels in supplier orders. Don't you?

Perhaps, should I also rename $showrefcustomerinsupplierline to $showrefinsupplierline !

As this function is also used by other document types such as supplier invoices, perhaps I need to take as basis of my reasoning  those other documents !